docs/orchestrating-elastic-stack-applications/logstash.asciidoc

```diff
@@ -367,6 +367,48 @@ spec:
 <4> Elasticsearch output definitions - use the environment variables created by the Logstash operator when specifying an `ElasticsearchRef`. Note the use of "normalized" versions of the `clusterName` in the environment variables used to populate the relevant fields.
 
 
+[id="{p}-logstash-external-es"]
+==== Connect to an external Elasticsearch cluster
+
+Logstash can connect to external Elasticsearch cluster that is not managed by ECK.
+You can reference a Secret instead of an Elasticsearch cluster in the `elasticsearchRefs` section through the `secretName` attribute:
+
+[source,yaml,subs="attributes,callouts"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: external-es-ref
+stringData:
+  url: https://abcd-42.xyz.elastic-cloud.com <1>
```

Comment:
I think we should add a note somewhere that we add the default port of `443` to `https` URLs, as this is different behavior to standard logstash plugins, and different to how non-cloud Elasticsearch works out of the box